### ISSUE TITLE
AccelerationStructureNV is not AccelerationStructureKHR

### ISSUE THREAD

**Issue by wumo** [KhronosGroup/Vulkan-Hpp#633]
On ubuntu 20.04 with nvidia-driver-440 installed which doesn't fully support vulkan 1.2, I create a `vk::UniqueAccelerationStructureNV` using the `vulkan-headers-1.2.140`. Segment fault happens when this `AccelerationStructureNV` is destoryed.
I found that : `VULKAN_HPP_DEFAULT_DISPATCHER.vkDestroyAccelerationStructureKHR`is `NULL` while `VULKAN_HPP_DEFAULT_DISPATCHER.vkDestroyAccelerationStructureNV` isn't. `vk::UniqueAccelerationStructureNV` always calls `vkDestroyAccelerationStructureKHR` to destory  itself which is the reason of segment fault.

```
// vulkan.hpp Line: 15726
using AccelerationStructureNV = AccelerationStructureKHR;
```

**Comment by wumo** [comment]
Currently I use custom deleter to destory AccelerationStructureNV:
```
std::unique_ptr<vk::AccelerationStructureNV, std::function<void(vk::AccelerationStructureNV *)>>(
    new vk::AccelerationStructureNV(as), [=](vk::AccelerationStructureNV *ptr) {
      device.destroyAccelerationStructureNV(*ptr);
      delete ptr;
    });
```

**Comment by asuessenbach** [comment]
This should have been resolved with #613, which unfortunately is based on VK_HEADER_VERSION 141.
As a workaround, you could adjust your local vulkan.hpp accordingly (look for the implementation of Device::destroy on AccelerationStructureKHR).

**Comment by wumo** [comment]
Tried  VK_HEADER_VERSION 141, it is indeeded fixed.

I think this issue can also be closed.
